cis194/week7/Sized.hs

```diff
@@ -23,6 +24,8 @@ instance Sized Size where
 instance Sized b => Sized (a,b) where
   size = size . snd
 
+instance Semigroup Size where
```

Comment:
> NOTE: Semigroup is a superclass of Monoid since base-4.11.0.0.

```
week7/zehua/src/Sized.hs:26:10-20: error:
    • No instance for (Semigroup Size)
        arising from the superclasses of an instance declaration
    • In the instance declaration for ‘Monoid Size’
   |
26 | instance Monoid Size where
   |          ^^^^^^^^^^^
```